### DISCUSSION TITLE
"Too many connections"

### DISCUSSION THREAD

**Discussion by alan345** [prisma/prisma#3429] [Q&A] [answered]
I am running prisma2 in production with AWS and RDS (MySQL).
After a few queries , Im receiving this error:

```
 "↵Invalid `prisma.user.findOne()` invocation in↵/home/ubuntu/nacho/prismaTs/src/utils.ts:70:36↵↵↵  Error occurred during query execution:↵ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Server(ServerError { code: 1040, message: "Too many connections", state: "HY000" })) })
```

In AWS, here you have the status of the databases:

![image](https://user-images.githubusercontent.com/15246526/91201481-55badc80-e6b5-11ea-9a06-05f1559f20bb.png)

this is my file: 
```

import { ApolloServer } from 'apollo-server-express'

const typeDefs = importSchema(path.join(__dirname, './schema.graphql'))

const schema = makeExecutableSchema({
  typeDefs,
  resolvers,
  directiveResolvers,
})
const prisma = new PrismaClient()

const server = new ApolloServer({
  debug: true,
  schema,
  validationRules: [depthLimit(8)],
  formatError: (error: any) => error,

  context: (req: any) => ({
    ...req,
    prisma,
  }),
})
```

I use a single instance of `prisma`
It was working fine with Prisma1, before the migration

related: https://github.com/prisma/prisma/issues/1983

**Comment by alan345** [comment, marked as answer]
My bad. I founded the error. 
I was on my end.

![image](https://user-images.githubusercontent.com/15246526/91206338-dbda2180-e6bb-11ea-9443-4251b0419a0e.png)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
